src/profile-logic/js-tracer.js

```diff
@@ -822,7 +822,7 @@ export function getSelfTimeSamplesFromJsTracer(
   // Each event type will have it's own timing information, later collapse these into
   // a single array.
   const { stringTable } = thread;
-  const samples = getEmptySamplesTable();
+  const samples = getEmptySamplesTableWithEventDelay();
```

Comment:
We don't push responsiveness values to that new samples object in the first place. So it looks okay to change it to eventDelay directly.